Source/Reporters/JunitXMLReporter.jsl

```diff
@@ -206,11 +208,11 @@ Define Class(
     Match(NItems(list_label),
       4,
       curr_suite = Trim(list_label[1]);
-      curr_test = Trim(list_label[2]) || "_" || Trim(list_label[3]) || "_" || Trim(list_label[4]);
```

Comment:
Remove dead code please.

Reply:
done in f91cfdac472b4c83635cb97fdb4b1a510082cd3a